### PR TITLE
Lineage pull request for: skeleton

### PR DESCRIPTION
**Pull request by cisagovbot**
# Lineage Pull Request #

[Lineage] has created this pull request to incorporate new changes found in an
upstream repository:

Upstream repository: [`https://github.com/cisagov/skeleton-tf-module.git`](https://github.com/cisagov/skeleton-tf-module.git)
Remote branch: `HEAD`

Check the changes in this pull request to ensure they won't cause issues with
your project.

------------

**Note:** *You are seeing this because one of this repository's maintainers has
configured [Lineage] to open pull requests.*

For more information:

🛠 [Lineage] configurations for this project are stored in `.github/lineage.yml`

📚 [Read more about Lineage][Lineage]

[//]: # (lineage:metadata:{&quot;slayed&quot;:true})
[Lineage]: https://github.com/cisagov/action-lineage/ "Lineage GitHub Action"

### CODE REVIEW

**Review by jsf9k — APPROVED** [review]
Nice job integrating the provider default tags jazz!

**Review by dav3r — CHANGES_REQUESTED** [review]
This PR needs to apply the late-breaking changes from https://github.com/cisagov/skeleton-tf-module/pull/51.

**Comment by mcdonnnj** [comment]
> 
> 
> This PR needs to apply the late-breaking changes from [cisagov/skeleton-tf-module#51](https://github.com/cisagov/skeleton-tf-module/pull/51).

@dav3r Was just waiting for the changes to propagate. They've pulled down now so it should be good-to-go.

File: examples/basic_usage/variables.tf [hunk 1]
```diff
@@ -32,3 +32,11 @@ variable "aws_region" {
   description = "The AWS region to deploy into (e.g. us-east-1)."
   default     = "us-east-1"
 }
+
+variable "tags" {
+  type        = map(string)
+  description = "Tags to apply to all AWS resources created."
+  default = {
+    Testing = true
+  }
```
Comment (jsf9k) [inline examples/basic_usage/variables.tf:41]: You can leave this as `default = {}` if you want.  In cisagov/skeleton-tf-module#51 I just wanted _something_ there so I could verify that the tags were being applied correctly.

File: main.tf [hunk 2]
```diff
@@ -34,21 +34,26 @@ data "aws_ami" "example" {
...
-    },
-  )
+  # The tag or tags specified here will be merged with the provider's
+  # default tags.
+  tags = {
+    "Name" = "Example"
+  }
+  # volume_tags does not yet inherit the default tags from the
+  # provider.  See hashicorp/terraform-provider-aws#19188 for more
+  # details.
```
Comment (jsf9k) [inline main.tf:54]: This file appears to be the example code from [cisagov/skeleton-tf-module](https://github.com/cisagov/skeleton-tf-module).  Do you intend to modify it to exercise your module?

**Review by jsf9k — CHANGES_REQUESTED** [review]
@mcdonnnj, is this code meant to be a root module or just a Terraform module?  If the former then the `tags` variable should be present in `variables.tf`.  If the latter then you need to update the example code in `examples/basic_usage` to use your module.

**Comment by mcdonnnj** [comment]
> 
> 
> @mcdonnnj, is this code meant to be a root module or just a Terraform module? If the former then the `tags` variable should be present in `variables.tf`. If the latter then you need to update the example code in `examples/basic_usage` to use your module.

@jsf9k This is effectively a placeholder repo right now. Since cloud native VDP scanning is on hold there is no actual Terraform for it currently. This was previously a direct clone of [cisagov/skeleton-tf-module](https://github.com/cisagov/skeleton-tf-module), but that caused it to miss any updates to that repo until I pushed https://github.com/cisagov/vdp-scanner-terraform/commit/83be81cd72ea50ed310498fb99b9f54bb355eb32 to update the [cisagov/action-lineage](https://github.com/cisagov/action-lineage) parent.

This PR is effectively it just pulling in all the updates to `cisagov/skeleton-tf-module` so that it's kept up-to-date until an actual implementation is begun.

**Comment by jsf9k** [comment]
> > @mcdonnnj, is this code meant to be a root module or just a Terraform module? If the former then the `tags` variable should be present in `variables.tf`. If the latter then you need to update the example code in `examples/basic_usage` to use your module.
> 
> @jsf9k This is effectively a placeholder repo right now. Since cloud native VDP scanning is on hold there is no actual Terraform for it currently. This was previously a direct clone of [cisagov/skeleton-tf-module](https://github.com/cisagov/skeleton-tf-module), but that caused it to miss any updates to that repo until I pushed [83be81c](https://github.com/cisagov/vdp-scanner-terraform/commit/83be81cd72ea50ed310498fb99b9f54bb355eb32) to update the [cisagov/action-lineage](https://github.com/cisagov/action-lineage) parent.
> 
> This PR is effectively it just pulling in all the updates to `cisagov/skeleton-tf-module` so that it's kept up-to-date until an actual implementation is begun.

OK, so long as you grok the difference between a module and a root module.

Given what you said, is there any point in keeping this thing around and having to take care of and feed it?  You could just recreate it when that work picks up again.

**Review by jsf9k — APPROVED** [review]
I get it now @mcdonnnj.

**Comment by mcdonnnj** [comment]
> 
> OK, so long as you grok the difference between a module and a root module.
> 
> Given what you said, is there any point in keeping this thing around and having to take care of and feed it? You could just recreate it when that work picks up again.

@jsf9k I'm fine with deleting it for now if we want to go that route.

**Review by dav3r — APPROVED** [review]
I'm good with whichever way you want to go on this @mcdonnnj.

**Pull request merged**